javascript/javascript2/week2/movie_function.js

```diff
@@ -0,0 +1,65 @@
+const movies = [{ "title": "'71", "year": 2014, "rating": 7.2, "votes": 41702, "running_times": 5940 }, { "title": "'A' gai wak", "year": 1983, "rating": 7.4, "votes": 11942, "running_times": 6300 }, { "title": "'Breaker' Morant", "year": 1980, "rating": 7.9, "votes": 10702, "running_times": 6420 }, { "title": "'Crocodile' Dundee II", "year": 1988, "rating": 5.5, "votes": 47180, "running_times": 6480 }]
+
+
+function findShortest(movies) {
+    return movies.reduce(function(prevWord, currWord) {
```

Comment:
Nice use of reduce! Reduce is a tricky method to learn so make sure to double check you've got the right solution here. It looks good from what I can see. See if you can refactor this to use an arrow function!